app/controllers/api_controller/accounts.rb

```diff
@@ -4,7 +4,7 @@ module Accounts
     # Accounts Subcollection Supporting Methods
     #
     def accounts_query_resource(object)
```

Comment:
I guess not, maybe you can delete it entirely in a sub-PR?  It looks like all of these are for translating resource names to internal method name where the resource name doesn't match the method but like you said, it seems this does, so perhaps we can ✂️ 🔥